Tests/Functional/Controller/Fixtures/TypoScript/Constants/PluginConfiguration.typoscript

```diff
@@ -0,0 +1,6 @@
+plugin.tx_tea {
+  persistence {
+    # cat=plugin.tx_tea//a; type=string; label=Default storage PID
```

Comment:
I guess the format is coming from ext configuration files and isn't needed here within the fixture?

Reply:
Done.